tests/data_classes.py

```diff
@@ -0,0 +1,157 @@
+import itertools
+import os
+import json
+import subprocess
+
+import libzfs
+
+
+class ZFS(libzfs.ZFS):
+
+    def __init__(self):
+        super().__init(history=True, history_prefix='<iocage>')
+        pools = [
+            p for p in self.pools
+            if self.zfs_get_property(p.name, 'org.freebsd.ioc:active')
+        ]
+        self.pool = pools[0] if pools else None
```

Comment:
We should probably throw an exception if they have multiple activated pools instead of blindly using the first in the tests. Especially since we wipe everything at the end, the user may not have intended that their production pool loses all their jails ;)